### PR TITLE
SLM's Draft Edits

### LINKED ISSUE #3: Draft Feedback
## Introduction
- "competible" should be changed to compatible 
- "a easy" should be changed to "an easy" in the last sentence
- "output" should be changed to "outputs" in the last sentence

## Progress
- Change to past tense
- remove "ready" from the first sentence

## Strategies

- ICA-AROMA "is", not "are"

## Results

- "Combined" acompcor doesn't refer to anat+temp, but to a combined WM+CSF anatomical mask (so it only applies to acompcor)

### PR DESCRIPTION
**Pull request by smeisler**
Related to issue #3 , but includes more.

### CODE REVIEW

File: report.md [hunk 1]
```diff
@@ -64,7 +64,12 @@ author:
   firstname: Hao-Ting
   email: wang.hao-ting@criugm.qc.ca
   affiliation: criugm, bsms
-  corref: criugm
+- initials: SLM
+  surname: Meisler
+  firstname: Steven Lee
+  email: smeisler@g.harvard.edu
+  affiliation: Harvard University, Massachusetts Institute of Technology
```
Comment (htwangtw) [inline report.md]: The template will take a tag for the institute. 
Can you update both your affiliations using the format at L44?
You can see how I set my affiliation as a reference

**Review by htwangtw — CHANGES_REQUESTED** [review]
Thanks for checking on the spellings etc! The only remaining thing to do is to make sure your affiliations are entered correctly and update your contribution :tada:

**Comment by smeisler** [comment]
Affiliations have been updated, where do I add contributions?

**Comment by htwangtw** [comment]
> Affiliations have been updated, where do I add contributions?

It's on L110 of the current version you are working on!

**Comment by smeisler** [comment]
done, thanks!

**Review by htwangtw — APPROVED** [review]
@smeisler you are a star
Thank you for the fast response and all the edits!

**Pull request merged**